agent/bls.mjs

```diff
@@ -810,8 +857,14 @@ async function do_sign_messages_impl(
     joExtraSignOpts,
     fn
 ) {
+    const imaState = state.get();
```

Comment:
Please, modify function so its maximum size will be 100 codelines:
https://github.com/skalenetwork/internal-support/blob/main/docs/wiki/code-guidelines.md#function-length

Reply:
Done